packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -1067,6 +1069,77 @@ function reappearLayoutEffectsOnFiber(node: Fiber) {
   }
 }
 
+function addOrRemovePendingBoundariesOnRoot(
```

Comment:
Can you rename this so it's clearer it's related to transitions? Maybe `commitTransitionProgress` or something